torch/_inductor/codegen/aot_inductor_interface.cpp

```diff
@@ -145,4 +146,31 @@ AOTInductorError AOTInductorModelContainerGetMaxOutputShape(
   })
 }
 
+AOTInductorError AOTInductorModelContainerSetConstants(
+  std::map<std::string, at::Tensor>& constantTensors) {
+
+    std::string currentFilePath = __FILE__;
```

Comment:
nit - it might be better to allow the user to specify where to store ```_constants.py```. It would make it more convenient for packaging the artifacts.